### ISSUE TITLE
Report test with commented out contents

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#537]
In case we have a test and the test contains only commented out code, we should issue a warning.

Following example should cause a warning:
```c#
[Test]
public void DoSomething()
{
/*
    some commented out code here, like
    Assert.That(xyz, Is.True);
*/
}
```
